library/VTKExtensions/Rendering/vtkF3DPolyDataMapper.cxx

```diff
@@ -265,18 +265,20 @@ bool vtkF3DPolyDataMapper::GetNeedToRebuildShaders(
   vtkOpenGLHelper& cellBO, vtkRenderer* ren, vtkActor* actor)
 {
   bool ret = this->Superclass::GetNeedToRebuildShaders(cellBO, ren, actor);
-  if (!ret)
+
+  // TODO: Put this logic in VTK
```

Comment:
add an issue to track this remark ?

Reply:
good point, I have identified many changes that should go in VTK, I will open an issue to track.

Reply:
https://github.com/f3d-app/f3d/issues/935